src/main/java/no/nav/familie/inntektsmelding/server/JettyServer.java

```diff
@@ -130,8 +130,7 @@ private static DataSource dataSource() {
         return new HikariDataSource(config);
     }
 
-    private void start() {
-        try {
+    private void start() throws Exception {
```

Comment:
Her er det vel like greit å kaste exception istedenfor å logge @mrsladek ? Hadde ikke oppstått i prod siden lansering

Reply:
Ok, jeg ser det er løst slik i fpsak så da kan vi vel anta at det holder for fpinntektsmelding også, så kan vi heller justere oss om vi ser det ikke dekker behovet.